addons/pos_self_order/views/res_config_settings_views.xml

```diff
@@ -17,7 +17,7 @@
                         </div>
                         <div class="d-flex flex-column align-items-start w-50">
                             <div class="d-flex flex-column align-items-start">
-                                <field name="pos_self_order_image_name"/>
+                                <field name="pos_self_order_image_name" invisible="1" />
```

Comment:
Remove this

Reply:
Ah no finally is needed to have invisible this field is used as filename just bellow